sdk/enter.go

```diff
@@ -80,6 +80,62 @@ func simpleChrootHelper(args []string) error {
 		return fmt.Errorf("Mounting %q failed: %v", newRepoRoot, err)
 	}
 
+	homedir := os.Getenv("HOME")
+	if homedir == "" {
+		return fmt.Errorf("HOME environment variable is not set.")
+	}
+
+	// mount the directory containing the ssh-agent socket in order
+	// to support a private repos during repo sync
+	sshAuthSock := os.Getenv("SSH_AUTH_SOCK")
```

Comment:
Using the same value of `$SSH_AUTH_SOCK` both inside and outside the chroot is what `enter_chroot.sh` does but it would probably be cleaner if inside the chroot the path was always bind to `/run/user/1000/ssh-XXXXXX` (create a randomly named temporary dir) and then update the value of `$SSH_AUTH_SOCK`

We should also match `enter_chroot.sh` by adding the equivilent of `mount -t tmpfs -o nosuid,nodev,mode=755 tmpfs /run` before dropping things into /run.